doc/changes/latest.inc

```diff
@@ -61,3 +62,4 @@ Bugs
 API changes
 ~~~~~~~~~~~
 - Deprecate arguments ``kind`` and ``path`` from :func:`mne.channels.read_layout` in favor of a common argument ``fname`` (:gh:`11500` by `Mathieu Scheltienne`_)
+- :func:`mne.warp_montage_volume` was deprecated in favor of :func:`mne.warp_montage` (doesn't require a volume) and :func:`mne.preprocessing.ieeg.make_montage_volume` (so that a volume of electrode contacts can still be made) (:gh:`11572` by `Alex Rockhill`_)
```

Comment:
Deprecated functions should be removed from the API ref doc and all refs to them changed to using code/double-backticks
```suggestion
- ``mne.warp_montage_volume`` was deprecated in favor of :func:`mne.warp_montage` (doesn't require a volume) and :func:`mne.preprocessing.ieeg.make_montage_volume` (so that a volume of electrode contacts can still be made) (:gh:`11572` by `Alex Rockhill`_)
```